MIDAS Journals.js

```diff
@@ -0,0 +1,122 @@
+{
+	"translatorID": "86c35e86-3f97-4e80-9356-8209c97737c2",
+	"label": "MIDAS Journals",
+	"creator": "Rupert Brooks",
+	"target": "(insight-journal|midasjournal|vtkjournal).org/browse/publication",
```

Comment:
What is in front this? Normally we have something like `^https?://(www)?`. Can we do this here also?

Moreover, escape the point in the regexp.